opencontext_py/apps/searcher/new_solrsearcher/README.md

```diff
@@ -0,0 +1,62 @@
+# README: Refactoring Search
```

Comment:
I figure that the first thing to do to understand this PR is to study this README.md